module/Application/src/Application/Module.php

```diff
@@ -23,17 +23,6 @@ public function onBootstrap(MvcEvent $e)
 
     public function getConfig()
     {
-        return include __DIR__ . '/config/module.config.php';
-    }
-
-    public function getAutoloaderConfig()
-    {
-        return array(
-            'Zend\Loader\StandardAutoloader' => array(
-                'namespaces' => array(
-                    __NAMESPACE__ => __DIR__ . '/src/' . __NAMESPACE__,
-                ),
-            ),
-        );
+        return include __DIR__ . '/../../config/module.config.php';
```

Comment:
@gianarb the config file is **required** here. If config file is not found, then it should raise a fatal error and stop the script execution.